indy_common/types.py

```diff
@@ -142,6 +143,54 @@ class GetContextField(MessageValidator):
     )
 
 
+class RsSchemaMetaField(MessageValidator):
+    schema = (
+        (RS_META_TYPE, ConstantField(RS_SCHEMA_META_TYPE)),
+        (RS_META_NAME, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
+        (RS_META_VERSION, VersionField(version_cls=RsMetaVersion)),
+    )
+
+
+class RsSchemaField(MessageValidator):
+    SCHEMA_IS_STRICT = False
+    schema = (
+        (RS_JSON_LD_TYPE, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
+        (RS_JSON_LD_ID, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
+    )
+    _base_types = None
+
+    def __init__(self, max_size=None, **kwargs):
+        if max_size is not None:
+            if not isinstance(max_size, int):
+                raise PlenumTypeError('max_size', max_size, int)
+            if not max_size > 0:
+                raise PlenumValueError('max_size', max_size, '> 0')
+        self.max_size = max_size
```

Comment:
Where `max_size` is used? Do we expect to add `_specific_validation` here in future?

Reply:
`max_size` is not used. I am not sure what `_specific_validation` is, but I did not plan to use it in the future.
I copied this code from context, for the size check, I will remove `self.max_size = max_size`, thanks.